content/de/cloud_computing.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Vorlage für Definition
+status: Feedback erwünscht
+category: Konzept
+---
+
+## Was es ist
+Das Konzept, hinter dem Cloud Computing ist die Bereitstellung von Systemressourcen wie CPU, Netzwerk und Speicher auf Abruf und über das Internet.
```

Comment:
Ich würde das dem weglassen. Und ich bin mir nicht sicher ob das Komma nach Konzept richtig ist. Oder ob es eher hinter `Cloud Computing` stehen muss.

Reply:
Komma muss in jedem fall weg, aber sonst passt es. Vorschlag: Wie wäre es mit "Das Konzept des Cloud Computing ist die ..."